pagopa/pn-frontend-e2e
# PN-9221-e-2-e-mittente-sezione-notifiche-ricerca-per-stato

## Short description
There isn't a test related to the **PN-9221** as you can see in the [confluence page](https://pagopa.atlassian.net/wiki/spaces/PN/pages/868778282/Mittente+-+Mapping+test-scenari)

## List of changes proposed in this pull request
- added new test file `011_2_ricercaPerCodiceStato.feature`
- added new feature step `Then Nella pagina piattaforma Notifiche è presente un campo di ricerca con un menu a tendina per selezionare lo stato della notifica`
- added related checks and methods


## How to test
Simply run `mvn clean test -Dcucumber.filter.tags=@TA_MittenteVerificaCampoStatoFiltroRicerca` and verify if the test works fine.
